stl/inc/execution

```diff
@@ -4235,7 +4235,7 @@ struct _Static_partitioned_transform_reduce2 { // transformed reduction task sch
             auto& _Transform_op = _This->_Transform_op;
             auto _Chunk         = _This->_Basis._Get_chunk(_Key);
             auto _Next          = _Chunk._First;
-            _Ty _Val{_Reduce_op(_Transform_op(*_Chunk._First), _Transform_op(*++_Next))};
+            _Ty _Val = _Reduce_op(_Transform_op(*_Chunk._First), _Transform_op(*++_Next));

```

Comment:
`=` needs to be aligned to make clang-format happy.
```suggestion
            _Ty _Val            = _Reduce_op(_Transform_op(*_Chunk._First), _Transform_op(*++_Next));
```

Reply:
Thank you for pointing it out, I used a different way to make both me and clang-format happy :-P

Reply:
Oh, I forgot to suggest that [GitHub's close/fix/resolve syntax](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) should be used so that the issue can be automatically closed by merging this PR.